Userland/Games/BrickGame/BrickGame.cpp

```diff
@@ -584,7 +584,7 @@ void BrickGame::paint_cell(GUI::Painter& painter, Gfx::IntRect rect, BrickGame::
     painter.fill_rect(rect, inside_color);
 }
 
-void BrickGame::paint_sidebar_text(GUI::Painter& painter, int row, StringView text)
+ErrorOr<void> BrickGame::paint_sidebar_text(GUI::Painter& painter, int row, StringView text)
```

Comment:
Why did you make this `ErrorOr<void>`? You're not returning any errors or `TRY`ing anything.

Reply:
Hmmm, I'm looking back at this now and I'm not sure. I must have gotten the idea that it was necessary and ended up conforming the fix to that. I'll push a fixup in just a minute